seedfarmer/mgmt/archive_support.py

```diff
@@ -90,17 +89,10 @@ def _process_archive(archive_name: str, response: Response, extracted_dir: str)
         archive_file.write(response.content)
 
     extracted_dir_path = os.path.join(parent_dir, extracted_dir)
-    embedded_dir = _extract_archive(archive_name, extracted_dir_path)
```

Comment:
If this is the only caller should we remove the return parameter?

Reply:
I would like to leave it for now for continuity sake? There are no assigned / unused variables.

Reply:
Doesn't need to be done for now, could be clean-up in the future.